tests/integration_test/action_service_test.py

```
@@ -14072,6 +14072,136 @@ def test_prompt_action_response_action_slot_prompt(mock_embedding, aioresponses)
+        payload={
+            'formatted_response': generated_text,
+            'response': {
+                'choices': [
+                    {'message': {'content': generated_text, 'role': 'assistant'}}
+                ]
+            }
+        }
+    )
+
+    # --- FIRE the webhook ---
+    request_object = json.load(open("tests/testing_data/actions/action-request.json"))
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)